src/ui/common/src/components/layouts/NavBar.tsx

```diff
@@ -84,10 +83,6 @@ const NavBar: React.FC<{
       ).length
   );
 
-  const handleUserPopoverClick = (event: React.MouseEvent) => {
```

Comment:
Why do we remove this?

Reply:
User popover has been removed and replaced with a settings gear. 

@likawind can you verify that the settings gear still opens when clicked?
If not, we may need to bring this function back.